utils/images_to_video.py

```diff
@@ -20,14 +22,19 @@ def images_to_video(images_paths, output_path, images_per_second=10, codec='MJPG
             max_width = width
         images.append(image)
 
-    resized_images = []
+    resize_ratios = []
     max_width_until_now = 0
-    resized_max_height = 0
     for image in images:
         _, width, _ = image.shape
         if width > max_width_until_now:
             max_width_until_now = width
-        resize_ratio = max_width / max_width_until_now
+        resize_ratios.append(max_width / max_width_until_now)
+
+    smooth_resize_ratios = np.convolve(resize_ratios, np.ones((zoom_out_smoothing,)) / zoom_out_smoothing, mode='valid')
```

Comment:
I don't get why you did this and what it does precisely, a little comment would help :)